src/test/unit/test.test.ts

```
@@ -1,9 +1,9 @@
-function sum(a: number, b: number) {
+export default function sum(a: number, b: number) {
   return a + b;
 }
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)